cactus/deployment/s3/engine.py

```diff
@@ -48,7 +48,7 @@ def _create_connection(self):
         aws_access_key, aws_secret_key = self.credentials_manager.get_credentials()
 
         return boto.connect_s3(aws_access_key.strip(), aws_secret_key.strip(),
-                               host=self._s3_api_endpoint, is_secure=self._s3_is_secure, port=self._s3_port,
```

Comment:
I think the changes in this file have nothing to do with the PR... +1 for the rest of the changes!